modules/client/src/interfaces.ts

```diff
@@ -259,20 +262,34 @@ export enum TransferType {
 export enum TokenType {
   NATIVE = "native",
   ERC20 = "erc20",
+  ERC721 = "erc721",
 }
 
 type BaseTokenTransfer = {
-  amount: bigint;
   creationDate: Date;
   transactionId: string;
+  to: string;
+  from: string;
 };
 
 type NativeTokenTransfer = BaseTokenTransfer & {
   tokenType: TokenType.NATIVE;
+  amount: bigint;
+  reference: string;
```

Comment:
reference has been removed. At the very, very least it should be optional... but it will be completely removed because no standard support it

Reply:
I was there on the sobgraph fot the native token transfers